content/collections/workflow/en/experiment-learnings.md

```diff
@@ -13,59 +13,113 @@ You’ve designed your experiment, rolled it out to your users, and given them e
 
-    An experiment is said to be **statistically significant** when we can confidently say that the results are highly unlikely to have occurred due to random chance. (More technically, it’s when we reject the null hypothesis.) That might sound pretty subjective, but it’s grounded solidly in statistics. Stat sig relies on a variant’s **p-value**, which is the probability of observing the data we see, assuming there is no difference between the variant and the control. If this probability drops below a certain threshold (statisticians refer to this threshold as the **alpha**), then we consider our experiment to have achieved statistical significance.
+## Analysis card
 
-    The *Summary* section will display a badge labeled *Significant* if stat sig was met, and a badge labeled *Not Significant* if stat sig was not met.
+At the top of the Analysis card is an overview that explains how your experiment performed, broken down by metric and variant. Below that, a collection of experiment results charts, which you can analyze by metric, display information about:
 
-    The *Summary* section may include multiple badges simultaneously:
+* Confidence intervals
+* Cumulative exposure*
+* Event totals
+* Mean value over time
 
-    * *Inconclusive*: the test was inconclusive for the primary metric.
-    * *Above Goal* or *Below Goal:* the primary metric's mean was either **above** or **below** its goal depending on the direction of the test (increase = above, decrease = below).
-    * *Above Control* or *Below Control:* the primary metric's mean was either **above** or **below** the control's mean, depending on the direction of the test (increase = above, decrease = below). These badges are only relevant to stat sig results.
+\* Cumulative exposure tracks the experiment as a whole.
 
-    ![summary.png](/docs/output/img/workflow/summary-png.png)
 
-2. At the top of the *Analysis* section is an overview of how your experiment performed, broken down by metric and variant. Below that is the experiment's **exposure definition:** how many variants were shown, what the primary metric was, and what the **exposure event** was. This is the event users will have to fire before being included in an experiment.  
-  
-    {{partial:admonition type='note'}}
-    The exposure event is **not the same thing** as the assignment event. If, for example, you’re running an experiment on your pricing page, a user might be evaluated on the home page for the experiment—but if they don’t visit the pricing page, they'll never actually be exposed to it. For that reason, this user should not be considered to be part of the experiment.  
-    {{/partial:admonition}}
+{{partial:admonition type="tip" heading="Chart filtering"}}
+The Experiment Results chart on the Activity tab responds to the selections you make in the [Filter card](#filter-card).
+{{/partial:admonition}}
+
+{{#
+We don't label the exposure event on this screen anywhere. Seems out of place in this article.
```

Comment:
@akhil-prakash This text around exposure events was already present in the article, but I couldn't find anything on the Analysis tab in an experiment. Unless I'm missing something, recommend removing / moving this to avoid confusion.

Reply:
this is super old. On the Activity card the only place the exposure event is shown is if you click on the "chart Controls" button